### DISCUSSION TITLE
Find: Recently used...

### DISCUSSION THREAD

**Discussion by Ciutnik** [devemux86/cruiser#302] [Q&A] [answered]
Many navigators allow you to quickly find recently used places (and routes). Maybe a long press on the "Search" button could be used to list a few recently used Favorites and POIs in navigation?

This is not an essential function, but it would facilitate routine activities.

**Comment by devemux86** [comment, marked as answer]
@Ciutnik thanks for the idea!

A history of recently searched items is already in my list.

Regarding routes / tracks, these can be covered by the upcoming big features:
- https://github.com/devemux86/cruiser/discussions/35
- https://github.com/devemux86/cruiser/discussions/81

**Reply by Ciutnik** [reply]
Thanks for the quick reply. I didn't want to mention about favorite (recent) routes, because I think implementing recently used points is probably easier 🙂 I thought of using a long press on "Search".

It's nice that you already thought about it, so all we have to do is wait patiently 🙂

**Reply by devemux86** [reply]
"Recently used points" seems to be something different than search history or favorite routes / tracks.

Do you probably mean, for example if we press some favorites / POI to use them in route planning,
then we could have a history list of such elements used in the past (and remembered in app restart)?

**Reply by Ciutnik** [reply]
Yes, that's exactly what I mean! 👍
